core/services/pipeline/runner.go

```diff
@@ -319,7 +319,7 @@ func (r *runner) InitializePipeline(spec Spec) (pipeline *Pipeline, err error) {
 }
 
 func (r *runner) run(ctx context.Context, pipeline *Pipeline, run *Run, vars Vars, l logger.Logger) TaskRunResults {
-	l = l.With("jobID", run.PipelineSpec.JobID, "jobName", run.PipelineSpec.JobName)
+	l = l.With("run.ID", run.ID, "executionID", uuid.New(), "run.PipelineSpecID", run.PipelineSpecID, "jobID", run.PipelineSpec.JobID, "jobName", run.PipelineSpec.JobName)
```

Comment:
Is `run.PipelineSpecID` a meaningful log key? We don't typically use that style. I see `specID` used other places.

Reply:
Yup, fixed it